radish34/api/src/graphql/resolvers/proposal.js

```diff
@@ -32,18 +34,19 @@ export default {
       input.sender = context.identity;
       const newProposal = await saveProposal(input);
       const proposal = newProposal.ops[0];
+      const { rfpId, type } = proposal;
       await saveNotice({
         resolved: false,
         category: 'proposal',
-        subject: `New Proposal: for RFP ${proposal.rfpId}`,
+        subject: `New Proposal: for RFP ${rfpId}`,
         from: currentUser.name,
         statusText: 'Pending',
         status: 'outgoing',
         categoryId: proposal.rfpId,
         lastModified: Math.floor(Date.now() / 1000),
       });
       pubsub.publish(NEW_PROPOSAL, { newProposal: proposal });
-      proposal.type = 'proposal_create';
+      type = 'proposal_create';
```

Comment:
This resolver attempts to set the `proposal.type` field to "proposal_create". By destructuring and setting `type = 'proposal_create';`, `proposal.type` never gets properly assigned a value